Config.cmake/test_fmi3.cmake

```diff
@@ -13,12 +13,6 @@
 
 include_directories(${FMIL_TEST_DIR}/FMI3)
 
-#Defines for the test FMUs
-set(FMU3_DUMMY_ME_MODEL_IDENTIFIER BouncingBall3) #This must be the same as in the xml-file
-set(FMU3_DUMMY_CS_MODEL_IDENTIFIER BouncingBall3) #This must be the same as in the xml-file
-set(FMU3_DUMMY_SE_MODEL_IDENTIFIER BouncingBall3) #This must be the same as in the xml-file
-set(FMU3_DUMMY_MF_MODEL_IDENTIFIER BouncingBall3_malformed) #This must be the same as in the xml-file
```

Comment:
Inlined these values - they were not understandable at all, were used as directory names, and pointed to BouncingBall and not fmu_dummy.